s3-sqs-lambda-terraform/example-pattern.json

```diff
@@ -0,0 +1,61 @@
+{
+  "title": "AWS Amazon S3 Lambda with SQS queue",
+  "description": "Create a Lambda function triggered for every jpg image file stored to S3 via SQS.",
+  "language": "node.js",
+  "level": "200",
+  "framework": "Terraform",
+  "introBox": {
+    "headline": "Create a Lambda function triggered for every jpg image file stored to S3 via SQS",
```

Comment:
The headline needs to stay below 75 characters, please reword

Reply:
Compressed and resumed now. Please check